src/test/visit_test_suite_run.sh.in

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+@PYTHON_EXECUTABLE@ @CMAKE_CURRENT_SOURCE_DIR@/visit_test_suite.py \
+   -d @CMAKE_BINARY_DIR@/testdata/  \
+   -b @CMAKE_CURRENT_SOURCE_DIR@/../../test/baseline/   \
+   -o output \
+   -e @CMAKE_BINARY_DIR@/bin/visit -n 1  $@
```

Comment:
- Why is -n 1 hard coded here? Doesn't that prevent running parallel tests if so desired?
- Would it make sense to name this script `runtest.in` so that the same command is executed in this dir as would have been before?
- In the spirit of above, why is `-o output` here? Isn't the default (`pwd`) ok?

Reply:
I added `-n 1` b/c the default is to use all procs, and we almost never want that. 

Maybe we should change the default in the test suite itself to be equiv to `-n 1`?

For `-o output`, we do need to pass it b/c the default will be relative to the python script (which is still in the source dir)

I am open to changing the name, as well -- but runtest.sh still exists, and all it does is call python visit_test_suite.py. Perhaps I should remove the old one?

Reply:
> Perhaps I should remove the old one?

👍